README.md

```diff
@@ -136,11 +146,26 @@ SELECT email FROM mytable WHERE name = 'Evil Bob'
 | evilbob@gmail.com |
 +-------------------+
 ```
+## Custom data source implementation
+
+To be able to create your own data source implementation you need to implement the following interfaces:
+
+- `sql.Database` interface. This interface will provide tables from your data source.
+  - If your database implementation supports add more tables, you might want to add support to `sql.Alterable` interface
```

Comment:
s/supports add more tables/supports adding more tables/
s/add support to/add support for/